internal/controller/defaults.go

```diff
@@ -28,13 +28,14 @@ import (
 )
 
 var imageDefaults = map[string]string{
-	"provisioner": "registry.k8s.io/sig-storage/csi-provisioner:v5.0.1",
-	"attacher":    "registry.k8s.io/sig-storage/csi-attacher:v4.6.1",
-	"resizer":     "registry.k8s.io/sig-storage/csi-resizer:v1.11.1",
-	"snapshotter": "registry.k8s.io/sig-storage/csi-snapshotter:v8.2.0",
-	"registrar":   "registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.11.1",
-	"plugin":      "quay.io/cephcsi/cephcsi:v3.13.1",
-	"addons":      "quay.io/csiaddons/k8s-sidecar:v0.12.0",
+	"provisioner":     "registry.k8s.io/sig-storage/csi-provisioner:v5.0.1",
+	"attacher":        "registry.k8s.io/sig-storage/csi-attacher:v4.6.1",
+	"resizer":         "registry.k8s.io/sig-storage/csi-resizer:v1.11.1",
+	"snapshotter":     "registry.k8s.io/sig-storage/csi-snapshotter:v8.2.0",
+	"odf-snapshotter": "quay.io/ocs-dev/csi-snapshotter:latest",
```

Comment:
Do we have this already pushed? Lets remember we should not break any existing tests/automation in other repo's

Reply:
Yes, we have this pushed already.
I think this is a placeholder for the image, and the configmap supplies the updated image that needs to be used, if present. So, we probably won't be breaking any automations.

Reply:
as we are adding it to CSV This features gets enabled by default, we need to update the consumers to first have the CRD's installed so that we are safe. Till that time we should keep this on hold?